apps/web/src/pages/templates/TemplatesListNoDataOld.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from '@emotion/styled';
```

Comment:
reverted the old view shown when there is no workflows, where we have two tiles: "Create Workflow" and "Try the Digest playground"